fastlane/Fastfile

```diff
@@ -30,6 +30,7 @@ ALL_LOCALES = [
   { glotpress: "vi", android: "vi",    google_play: "vi",     promo_config: {} },
   { glotpress: "zh-cn", android: "zh-rCN", google_play: "zh-CN",  promo_config: {} },
   { glotpress: "zh-tw", android: "zh-rTW", google_play: "zh-TW",  promo_config: {} },
+  # From this point are locales that are sto;; used for downloading `strings.xml`… but not for release notes – and thus don't need a `google_play` key. See `RELEASE_NOTES_LOCALES` below.
```

Comment:
Whoops, seems my fingers shifted 1 key to the right when I typed that comment (`still` -> `sto;;`)… 😅